src/lib/rsshub.ts

```diff
@@ -78,7 +78,7 @@ export function getPageRSSHub(data: {
 
     let parsedDomain;
     try {
-        parsedDomain = psl.parse(new URL(url).hostname);
+        parsedDomain = parse(new URL(url).hostname);
```

Comment:
Note: https://github.com/salesforce/tough-cookie/pull/346#discussion_r1429217096

Some of these might be relevant depending on your use-case. There are slight differences in default behavior between psl and tldts.

Reply:
Thank you so much, private domains will not be used in our extension, so there is no problem.

Reply:
If you want to shrink the bundle size a bit, there is a version of tldts that only ships with ICANN rules:  https://github.com/remusao/tldts/tree/master/packages/tldts-icann#tldts---blazing-fast-url-parsing-icann-rules-only